gammapy/cube/tests/test_counts.py

```diff
@@ -82,3 +84,21 @@ def test_fill_map_counts_keyerror(events):
     cntmap = WcsNDMap.create(binsz=0.1, npix=10, axes=[axis])
     with pytest.raises(KeyError):
         fill_map_counts(cntmap, events)
+
+@requires_data('gammapy-extra')
+def test_fill_map_counts_multiple_energy_axes(events):
+    axis_mc = MapAxis([0, 2, 4], name='energy_mc', unit='TeV')
+    axis_reco = MapAxis([0.001, 1000.], name='energy_reco', unit='TeV')
+    axis = MapAxis([0.001, 1000.], name='energy', unit='TeV')
+
+    cntmap1 = WcsNDMap.create(binsz=1, npix=10, coordsys='GAL', axes=[axis, axis_mc])
+    cntmap2 = WcsNDMap.create(binsz=1, npix=10, coordsys='GAL', axes=[axis_mc, axis_reco])
+
+    col_mc = Column(np.ones(len(events.table)) * u.TeV, name='ENERGY_MC')
+    events.table.add_column(col_mc)
```

Comment:
You can just do this, and remove the Column import at the top:
```
events.table['ENERGY_MC'] = 1 * u.TeV
```
It makes the same addition to the table.

Reply:
Nice.